.goreleaser.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 
 release:
   github:
-    owner: nats-io
+    owner: '{{ envOrDefault "GITHUB_REPOSITORY_OWNER" "nats-io" }}'
```

Comment:
Is that essentially so that you can override in your fork repo to test? (and if so, that's great and not asking to remove, just checking)

Reply:
Yep, just means that forking to `neilalexander/nats-server` doesn't need more fudging to work.